src/main/java/com/redhat/devtools/lsp4ij/LanguageServiceAccessor.java

```diff
@@ -584,4 +586,31 @@ private void disposeAllServers() {
         }
     }
 
+    /**
+     * Returns the modification trackers for all language server definitions for the file.
+     *
+     * @param file the PSI file
+     * @return all relevant modification trackers
+     */
+    @NotNull
+    @ApiStatus.Internal
+    public Set<ModificationTracker> getModificationTrackers(@NotNull PsiFile file) {
+        Set<ModificationTracker> modificationTrackers = new LinkedHashSet<>();
+
+        // Add the registry-level modification tracker
+        modificationTrackers.add(LanguageServersRegistry.getInstance().getModificationTracker());
+
+        // Add modification trackers for all of the file's language server definitions
+        VirtualFile virtualFile = file.getVirtualFile();
+        if (virtualFile != null) {
+            MatchedLanguageServerDefinitions mappings = getMatchedLanguageServerDefinitions(virtualFile, project, true);

```

Comment:
Gets all matched language server definitions for the file regardless of their status.